ProcessLib/GroundwaterFlow/GroundwaterFlowProcess.h

```diff
@@ -69,9 +70,12 @@ class GroundwaterFlowProcess final : public Process
             MeshLib::addPropertyToMesh(*_balance_mesh, _balance_pv_name,
                                        MeshLib::MeshItemType::Cell, 1,
                                        init_values);
+            const int process_id = 0;
```

Comment:
This I don't understand: on one side process id is 0 but the passed function's argument is not used.... I'd expect that the callee is deciding which `process_id` is being used...

Reply:
Added a comment.